README.md

```diff
@@ -1 +1,13 @@
 # hw01
+
+##*Ta'er*, Seer of Paths, Traverser of Seas
+All hail the great *Ta'er*, Lord of the Storm! Raised as Seth Markow in the humble coastal city of *Saun Dieagoh*, *Ta'er* began preaching his gospel in the tundra city of *Shikagho*, and soon his teachings became law, and spread across the world, ushering in a new era of peace and prosperity for the people of Earth. Now, as we venture further and further into the outer reaches of space, soon others, too, will know the goodness of *Ta'er*, Knower of the Way, Seer of the Future!
```

Comment:
First of all - I love Dune (it is actually also my mom's favorite book)! This intro is super fun and has good use of emphasis with the italics!

Reply:
My father-in-law would love this. Huge fan of Dune. He indoctrinated his children with the 80s movie at an early age.

![](https://media.giphy.com/media/FK96Gm1RMRGi4/giphy.gif)

I tried watching it with my in-laws a year ago and the only thing I can remember thinking is why is Picard here? I am hopeful the new movie is better, though I've never tried reading the books.

Reply:
Also, Sting. Why Sting?